Package.swift

```diff
@@ -0,0 +1,9 @@
+import PackageDescription
+
+let package = Package(
+    name: "GRDB.swift",
```

Comment:
Is the name of the package also used when one imports it from Swift with `import` ? Wouldn't be `GRDB` a better name, then? (Honest question, I really don't know)